serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -18,6 +18,8 @@
 ** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc[Kogito Serverless Workflow Tools extension in Quarkus Dev UI]
 *** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-instances-page.adoc[Workflow Instances in Kogito Serverless Workflow Tools extension]
 *** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc[Workflow Definitions in Kogito Serverless Workflow Tools extension]
+** xref:tooling/kn-plugin-workflow/kn-plugin-workflow-overview.adoc[Workflow plugin for Knative CLI]
```

Comment:
@AjayJagan @ljmotta I missed this during my review. I think we can keep this guide on this path instead. We don't need to add an extra hoop.

```suggestion
** xref:tooling/kn-plugin-workflow-overview.adoc[Workflow plugin for Knative CLI]
```

Can you please move?

Reply:
done :+1: